src/utils/compose_ts.ts

```diff
@@ -491,6 +491,7 @@ async function qemuTransposeBuildStream({
 	task.buildStream = (await transpose.transposeTarStream(
 		task.buildStream,
 		transposeOptions,
+		// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
 		dockerfilePath || undefined,
```

Comment:
```suggestion
		dockerfilePath,
```

I don't think we need a fallback here at all since `dockerfilePath` is already typed as `string | undefined`